namesrv/src/main/java/org/apache/rocketmq/namesrv/NamesrvStartup.java

```diff
@@ -42,7 +42,8 @@
 
 public class NamesrvStartup {
 
-    private static Logger log;
+    private final static Logger LOGGER = LoggerFactory.getLogger(LoggerName.NAMESRV_LOGGER_NAME);
+    private final static Logger LOGGER_CONSOLE = LoggerFactory.getLogger(LoggerName.NAMESRV_CONSOLE_LOGGER_NAME);
```

Comment:
why not use LOGGER ?

Reply:
@fuyou001 LOGGER print log in file, LOGGER_CONSOLE print on console, -p command should be printed on console

Reply:
keep log format may better

https://docs.oracle.com/javase/specs/jls/se17/html/jls-4.html

https://stackoverflow.com/questions/1417190/should-a-static-final-logger-be-declared-in-upper-case/1417216#1417216

Reply:
> @fuyou001 LOGGER print log in file, LOGGER_CONSOLE print on console, -p command should be printed on console

I got it

Reply:
> keep log format may better
> 
> https://docs.oracle.com/javase/specs/jls/se17/html/jls-4.html
> 
> https://stackoverflow.com/questions/1417190/should-a-static-final-logger-be-declared-in-upper-case/1417216#1417216

@lizhimins thanks for you suggestion. I hava read docuemnts and I will modify it later.